docs/getting-started/setting-up-a-local-chain.md

```diff
@@ -0,0 +1,139 @@
+
+## Prerequisites
+
+- **`gnoland` set up. Reference the [Installation](installation.md#3-installing-other-gno-tools) guide for steps**
+
+## Starting a node with a default configuration
+
+You can start a Gno blockchain node with the default configuration by navigating to the `gno.land` sub-folder and
+running the following command:
+
+```bash
+gnoland start
+```
+
+The command will trigger a chain initialization process (if you haven't run the node before), and start the Gno node,
+which is ready to accept transactions and interact with other Gno nodes.
+
+![gnoland start](../assets/getting-started/setting-up-a-local-chain/gnoland-start.gif)
+
+To view the command defaults, simply run the `help` command:
+
+```bash
+gnoland start --help
+```
+
+Let's break down the most important default settings:
+
+- `chainid` - the ID of the Gno chain. This is used for Gno clients, and distinguishing the chain from other Gno
+  chains (ex. through IBC)
+- `config` - the custom node configuration file
+  for more details on utilizing this file
+- `genesis-balances-file` - the initial premine balances file, which contains initial native currency allocations for
+  the chain. By default, the genesis balances file is located in `gno.land/genesis/genesis_txs.txt`, this is also the
+  reason why we need to navigate to the `gno.land` sub-folder to run the command with default settings
+- `root-dir` - the working directory for the node configuration and node data (state DB)
+
+:::info Resetting the chain
+
+As mentioned, the working directory for the node is located in `root-dir`. To reset the chain, you need
+to delete this directory and start the node up again. If you are using the default node configuration, you can run
+`make fclean` from the `gno.land` sub-folder to delete the `tempdir` working directory.
+
+:::
+
+## Changing the chain ID
+
+:::info Changing the Gno chain ID has several implications
+
+- It affects how the Gno node communicates with other Gno nodes / chains
+- Gno clients that communicate through JSON-RPC need to match this value
+
+:::
+
+To change the Gno chain ID, run the following command:
+
+```bash
+gnoland start --chainid NewChainID
+```
+
+We can verify the chain ID has been changed, by fetching the status of the node and seeing the
+associated chain ID. By default, the node exposes the JSON-RPC API on `http://127.0.0.1:26657`:
+
+```bash
+curl -H "Content-type: application/json" -d '{
+    "jsonrpc": "2.0",
+    "method": "status",
+    "params": [],
+    "id": 1
+}' 'http://127.0.0.1:26657'
+```
+
+We should get a response similar to this:
+
+```json
+{
+  "jsonrpc": "2.0",
+  "id": 1,
+  "result": {
+    "node_info": {
+      "version_set": [
+        // ...
+      ],
+      "net_address": "g10g9r37g9xa54a6clttzmhk2gmdkzsntzty0cvr@0.0.0.0:26656",
+      "network": "NewChainID"
+      // ...
+    }
+  }
+}
+```
+
+:::danger Chain ID can be set only once
```

Comment:
We should provide a more comprehensive and global explanation because many configuration (not only chainID) cannot be changed or may not be meaningful to change after connecting to other peers.

Something like: "It's important to configure your node properly before launching it in a distributed network. Keep in mind that changes may not be applicable once connected."

Reply:
@leohhhn @waymobetta just a ping to keep this somewhere in the back of our minds for future docs efforts (detailing the configuration).

I have updated this disclaimer in the meantime, for this PR:
[ef941e0](https://github.com/gnolang/gno/pull/1046/commits/ef941e01a455b8977d5e7e74a6ef25d1ee341820)